src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractReportServiceImpl.java

```diff
@@ -477,6 +480,17 @@ public void setReportValidationErrorsSubTitle(String reportValidationErrorsSubTi
         this.reportValidationErrorsSubTitle = reportValidationErrorsSubTitle;
     }
 
+    public String getReportValidationErrorsSubTitleByPassedTransactionNote() {
+        if (StringUtils.isEmpty(reportValidationErrorsSubTitle)) {
+            setReportValidationErrorsSubTitle(ConcurConstants.StandardAccountingExtractReport.SAE_VALIDATION_SUB_REPORT_BYPASSED_NOTE_NOT_SET_IN_CONFIGURATION_FILE);
```

Comment:
Should this be calling the setter for "reportValidationErrorsSubTitleByPassedTransactionNote" instead?

Reply:
erg, yep, thanks